.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+# Release and deploy
```

Comment:
Would you mind adding more detail comment to this file? Like this .yml will switch between dirs and run scripts such as `/scripts/deploy.js`. Since this file is really important.

Reply:
I can only add comments on what this yml is for in general and why we need it. I don't think is necessary to be detailed to dirs or scripts which could be overcoupling.